labelImg.py

```diff
@@ -1196,10 +1197,15 @@ def show_bounding_box_from_annotation_file(self, file_path):
         else:
             xml_path = os.path.splitext(file_path)[0] + XML_EXT
             txt_path = os.path.splitext(file_path)[0] + TXT_EXT
+            json_path = os.path.splitext(file_path)[0] + JSON_EXT
+
             if os.path.isfile(xml_path):
                 self.load_pascal_xml_by_filename(xml_path)
             elif os.path.isfile(txt_path):
                 self.load_yolo_txt_by_filename(txt_path)
+            elif os.path.isfile(txt_path):
```

Comment:
```
    elif os.path.isfile(txt_path):
                self.load_yolo_txt_by_filename(txt_path)
    elif os.path.isfile(txt_path):
                self.load_create_ml_json_by_filename(json_path, file_path)
```
The second `elif` will never happen

Reply:
fixed this, Last elif should be 
` elif os.path.isfile(json_path):
                self.load_create_ml_json_by_filename(json_path, file_path)`